lib/tasks/build.js

```diff
@@ -1,11 +1,15 @@
 'use strict';
 
 var runCommand = require('../utils/run-command');
+var Promise    = require('../ext/promise');
```

Comment:
'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.<br>'require' is not defined.